css/override.css

```diff
@@ -17,6 +17,22 @@ blockquote {
     border-left: 2px solid #4D4D4D;
 }
 
+blockquote-inverse {
+    padding: 0 50px 0 200px;
+    margin-right: 0;
+    border-right: 2px solid #4D4D4D;
+    text-align: right;
+}
+
+footer {
+    text-align: left;
```

Comment:
These usual types of styling are not needed as they are available in bootstrap https://getbootstrap.com/docs/4.1/utilities/text/#text-alignment